api/app.models.js

```diff
@@ -6,3 +6,28 @@ exports.getCategory = () => {
     return categories.rows;
   });
 };
+exports.getReviewById = (id) => {
+  return db
+    .query("SELECT * FROM reviews WHERE review_id = $1;", [id])
+    .then((review) => {
+      return review.rows;
+    });
+};
+exports.getReview = (sort_by = "created_at") => {
+  const validSortQueries = [
+    "created_at",
+    "votes",
+    "comment_count",
+    "review_id",
+  ];
+  if (!validSortQueries.includes(sort_by)) {
+    return Promise.reject({ status: 400, msg: "Invalid sort query!" });
+  }
+  return db
+    .query(
+      "SELECT r.review_id, r.title, r.category, r.designer, r.owner, r.review_img_url, r.created_at, r.votes, COUNT(c.comment_id) AS comment_count FROM reviews AS r LEFT JOIN comments AS c ON r.review_id = c.review_id GROUP BY r.review_id ORDER BY created_at DESC"
```

Comment:
Nice sql query, good to see you've used the COUNT aggregate function to help you count the comments, well done!